lte/gateway/c/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -483,6 +481,7 @@ static void* mme_app_thread(__attribute__((unused)) void* args) {
 
   // Service started, but not healthy yet
   send_app_health_to_service303(&mme_app_task_zmq_ctx, TASK_MME_APP, false);
+  start_stats_timer();
```

Comment:
This should not started in the main thread right? Can you check in master?

Reply:
It is started on the main thread: https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/tasks/mme_app/mme_app_main.c#L522

Reply:
This is not the main thread.